src/module-loader.ts

```diff
@@ -79,7 +80,32 @@ export default class ModuleLoader {
     let isESM: boolean | undefined
     try {
       ({isESM, filePath} = ModuleLoader.resolvePath(config, modulePath))
-      const module = isESM ? await import(url.pathToFileURL(filePath).href) : require(filePath)
+      const module = isESM ? await import(pathToFileURL(filePath).href) : require(filePath)
```

Comment:
meta: this whole static-only class bothers me.  You could have a 2 exported functions (`load``, `loadWithData`, `loadWithDataFromManifest`) and then some non-exported ones.

By breaking these up, you'd have the opporunity to only `require/import` what's needed instead of doing the whole "class" 

And this isn't a top-level export, but that would prevent poeple from using functions like `resolvePath` outside of this.